.circleci/config.yml

```diff
@@ -53,37 +53,34 @@ install_dep_16: &install_dep_16
       name: Install Dependencies
       command: |
         sudo apt-get install -y libopenmpi-dev
-        pip install --progress-bar off torch==1.6.0+cu101 -f https://download.pytorch.org/whl/torch_stable.html
         pip install --progress-bar off -r requirements-test.txt
+        pip install --progress-bar off torch==1.6.0+cu101 -f https://download.pytorch.org/whl/torch_stable.html
         pip install --progress-bar off git+https://github.com/msbaines/torch_pg.git@c85c96f#egg=torch-pg
         python -c 'import torch; print("Torch version:", torch.__version__)'
         python -m torch.utils.collect_env
 
-install_dep_17_cpu: &install_dep_17_cpu
+
+install_dep_17: &install_dep_17
   - run:
       name: Install Dependencies
       command: |
         sudo apt-get install -y libopenmpi-dev
-        pip install --progress-bar off torch==1.7.0+cu101 -f https://download.pytorch.org/whl/torch_stable.html
         pip install --progress-bar off -r requirements-test.txt
```

Comment:
that's what I was trying to explain @msbaines and @min-xu-ai , 1.5 and 1.6 were never actually running because of that. This line updated torch because of the torchvision dependency, and installed a version not compatible with CUDA on the agents (11). Thus we were initially not testing cuda, and not testing 1.5 and 1.6. 

This was a hotfix to at least make sure that CUDA was running with 1.7 https://github.com/facebookresearch/fairscale/pull/267, and I thought at the time that https://github.com/facebookresearch/fairscale/pull/251 would handle the rest